system/file.ts

```diff
@@ -33,11 +33,6 @@ export class File {
   // The string limit per Preload call.
   private static preloadLimit = 259;
 
-  // eslint-disable-next-line no-useless-constructor
-  private constructor() {
-    // nothing
-  }
-
```

Comment:
Removing this allows the File class to be instantiated, which is not intended behavior.